README.md

```diff
@@ -7,9 +7,10 @@
 [![CI Status](https://github.com/babybuddy/babybuddy/actions/workflows/ci.yml/badge.svg)](https://github.com/babybuddy/babybuddy/actions/workflows/ci.yml)
 [![Coverage Status](https://coveralls.io/repos/github/babybuddy/babybuddy/badge.svg?branch=master)](https://coveralls.io/github/babybuddy/babybuddy?branch=master)
 [![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/babybuddy/babybuddy)
```

Comment:
Would any of this stuff be advantageous for Gitpod as well? I don't have a strong opinion about including this... just curious if there is an easy improvement here as well.

Reply:
I like gitpod, but I lost track of it after GitHub codespaces came out. We could maybe get gitpod to use the devenv config if you want me to look into it. I think they are competitors to be honest.